packages/mgt-components/src/components/mgt-person-card/mgt-person-card.ts

```diff
@@ -615,25 +647,46 @@ export class MgtPersonCard extends MgtTemplatedComponent {
 
     const currentSectionIndex = this._currentSection ? this.sections.indexOf(this._currentSection) : -1;
 
-    const navIcons = this.sections.map((section, i, a) => {
+    const additionalSectionTemplates = this.sections.map((section, i) => {
+      let name = section.tagName.toLowerCase();
       const classes = classMap({
         active: i === currentSectionIndex,
         'section-nav__icon': true
       });
       return html`
-        <button class=${classes} @click=${() => this.updateCurrentSection(section)}>${section.renderIcon()}</button>
+        <fluent-tab id="${name}-Tab" class=${classes}
+          slot="tab" @click=${() => this.updateCurrentSection(section)}>${section.renderIcon()}
+        </fluent-tab>
+      `;
+    });
+
+    const additionalPanelTemplates = this.sections.map((section, i) => {
+      return html`
+        <fluent-tab-panel slot="tabpanel">
+              <div class="inserted">${this._currentSection ? section.asFullView() : null}</div>
+        </fluent-tab-panel>
       `;
     });
 
     const overviewClasses = classMap({
       active: currentSectionIndex === -1,
-      'section-nav__icon': true
+      'section-nav__icon': true,
+      overviewTab: true
     });
+
     return html`
-      <button class=${overviewClasses} @click=${() => this.updateCurrentSection(null)}>
-        ${getSvg(SvgIcon.Overview)}
-      </button>
-      ${navIcons}
+            <fluent-design-system-provider use-defaults>
```

Comment:
the design system provider is no longer necessary in the latest versions of fluent ui

Reply:
Thanks, Working on it: 
0d567f2

I'm having some trouble with the css application without the `system-provider`, especially with the tabs. I need some more investigation to figure out the cause.